test/tests/vectorpostprocessors/mesh_division_functor_reduction/reduction.i

```diff
@@ -0,0 +1,108 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 20
+  ny = 10
+  # This ensures the CSV VPP output remains ordered the same way with distributed meshes
+  allow_renumbering = false
+[]
+
+[MeshDivisions]
+  [mesh_div]
+    type = CylindricalGridDivision
+    n_radial = 4
+    n_azimuthal = 4
+    axis_direction = '0 0 1'
+    azimuthal_start = '1 0 0'
+    center = '0.5 0.5 0'
+    r_max = 2
+  []
+[]
+
+[Functions]
+  [u_fn]
+    type = ParsedFunction
+    expression = 'x + 2 * if(y > 0.5, y, 0)'
+  []
+[]
+
+[AuxVariables]
+  [u]
+  []
+  [u_fv]
+    type = MooseVariableFVReal
+  []
+[]
+
+[ICs]
+  [u_ic]
+    type = FunctionIC
+    variable = u
+    function = u_fn
+  []
+  [u_fvic]
+    type = FunctionIC
+    variable = u_fv
+    function = u_fn
+  []
```

Comment:
I thought this wasn't supposed to work anymore after @grmnptr's changes but clearly it does

Reply:
Yeah for @GiudGiud 's request this is still not erroring. It will error for the linear fv variable.

Reply:
If it comes down to it, we could ban it. But I would prefer it kept working if we can